docs/test/replication-controller-list.md

```diff
@@ -0,0 +1,42 @@
+|--------------------------------------|------------------------------|-------------------------------------|-----------------|--------------|-----------------|---------------------------|------------|--------|---------------------------------------------------------------------------------------------|
+| Replication controller count on page | 1                            | odd                                 | even            | 20-30        |                 |                           |            |        |                                                                                             |
+| Screen size                          | normal                       | mobile                              | narrow          | wide         | active resizing |                           |            |        |                                                                                             |
+| Replication controller status        | valid state                  | error state                         |                 |              |                 |                           |            |        | error increases height of the card, error state can be caused by setting non-existing image |
+| Replication controller fields        | normal                       | max length                          | hight pod count |              |                 |                           |            |        |                                                                                             |
+| Replication controller labels        | normal                       | long text (253 prefix and 63 label) | 1               | odd          | even            | short and long text mixed | 20-30      |        | labels increase height of the card                                                          |
+| Actions                              | view details                 | edit pod count                      | delete          | display logs |                 |                           |            |        |                                                                                             |
+| Edit pod count                       | negative                     | 0                                   | 1               | less than 10 | more than 30    | empty                     | scale down | cancel |                                                                                             |
+| Delete replication controller action | normal                       | with checkbox checked               | cancel          |              |                 |                           |            |        |                                                                                             |
```

Comment:
Yes, but if we don't want to combine factors then there are only two possibilities - checked or not.  We must remember, that this factor will be combined with other ones. There is a possibility of adding factor telling if replication controllers have services or not, but it's not visible from this view... It's very problematic from my point of view.